scss/_custom.scss

```diff
@@ -245,54 +245,56 @@ $transition-collapse:    height .35s ease !default;
 //
 // Font, line-height, and color for body text, headings, and more.
 
-$font-family-sans-serif: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif !default;
+$font-family-sans-serif: "Helvetica Neue", Arial, Roboto, sans-serif !default;
 $font-family-monospace:  Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace !default;
 $font-family-base:       $font-family-sans-serif !default;
 
-$font-size-base: 1rem !default; // Assumes the browser default, typically `16px`
-$font-size-lg:   1.25rem !default;
-$font-size-sm:   .875rem !default;
-$font-size-xs:   .75rem !default;
+$font-size-base: .9rem;
+$font-size-lg:   1.1rem;
+$font-size-sm:   .75rem;
+$font-size-xs:   .7rem;
 
-$font-weight-normal: normal !default;
-$font-weight-bold: bold !default;
+$font-weight-light: 300;
+$font-weight-normal: 400;
+$font-weight-semibold: 500;
+$font-weight-bold: 700;
 
-$font-weight-base: $font-weight-normal !default;
-$line-height-base: 1.5 !default;
+$font-weight-base: $font-weight-light;
+$line-height-base: 2;
 
 $font-size-h1: 2.5rem !default;
 $font-size-h2: 2rem !default;
-$font-size-h3: 1.75rem !default;
-$font-size-h4: 1.5rem !default;
-$font-size-h5: 1.25rem !default;
-$font-size-h6: 1rem !default;
+$font-size-h3: 1.5rem;
+$font-size-h4: 1.1rem;
+$font-size-h5: 1rem;
+$font-size-h6: .95rem;
 
 $headings-margin-bottom: ($spacer / 2) !default;
 $headings-font-family:   inherit !default;
-$headings-font-weight:   500 !default;
+$headings-font-weight:   $font-weight-bold;
 $headings-line-height:   1.1 !default;
 $headings-color:         inherit !default;
 
-$display1-size: 6rem !default;
-$display2-size: 5.5rem !default;
-$display3-size: 4.5rem !default;
-$display4-size: 3.5rem !default;
+$display1-size: $font-size-h1;
+$display2-size: $font-size-h2;
+$display3-size: $font-size-h3;
+$display4-size: $font-size-h4;
 
-$display1-weight:     300 !default;
-$display2-weight:     300 !default;
-$display3-weight:     300 !default;
-$display4-weight:     300 !default;
+$display1-weight:     $font-weight-light;
+$display2-weight:     $font-weight-light;
+$display3-weight:     $font-weight-light;
+$display4-weight:     $font-weight-light;
 $display-line-height: $headings-line-height !default;
 
-$lead-font-size:   1.25rem !default;
-$lead-font-weight: 300 !default;
+$lead-font-size:   $font-size-lg;
+$lead-font-weight: $font-weight-light;
 
-$small-font-size: 80% !default;
+$small-font-size: .65rem;
```

Comment:
:) we can do 80% as well — I am good with that. it is engineer vs designer mindset. 

Given you comment I gave it more thought and understand why one would want a utility `small` class in order to quickly reduce text size as needed. it makes sense.

I was defining a full typographic scale that we would adhere very strictly across the site. 6 header sizes + 5 copy sizes from lead to body, to 3 helper caption sizes. Less flexibility, strict guidelines, no undefined type size use.

- 

**I am comfortable with both.** if we make small a % defined class this effectively doubles the amount of type sizes across the size. This can be both good and bad depending on what we need. More flexible, but easier for the design to fall apart, without a watchful eye.